src/common/config.go

```diff
@@ -75,6 +75,8 @@ type FeaturesConfig struct {
 	Import_cache        string `json:"import_cache"`
 	Downsize_paused_mem bool   `json:"downsize_paused_mem"`
 	Enable_seccomp      bool   `json:"enable_seccomp"`
+	Warmup              bool   `json:"warmup"`
+	COW                 bool   `json:"COW"`
```

Comment:
What does COW do?  At a glance, it seems like it determines whether you use Zygotes to create new sandboxes.  If you don't want to use Zygotes, you can just choose not to have a Zygote tree by configuring the "Import_cache" setting ahead.  So I'm wondering whether it's necessary to have a second config option?

Reply:
in cow we'll still have every zygotes in the tree, but they don't share the memory anymore. Let's say we have a tree root->A->[A,B], if cow is disabled, it will create a sandbox import A and another sandbox import A.B separately.